hack/util.sh

```diff
@@ -443,14 +448,14 @@ reset_tmp_dir() {
 	local sudo="${USE_SUDO:+sudo}"
 
 	set +e
-	${sudo} rm -rf ${BASETMPDIR} &>/dev/null
+	${sudo} rm -rf "${BASETMPDIR}" &>/dev/null
 	if [[ $? != 0 ]]; then
 		echo "[INFO] Unmounting previously used volumes ..."
-		findmnt -lo TARGET | grep ${BASETMPDIR} | xargs -r ${sudo} umount
-		${sudo} rm -rf ${BASETMPDIR}
+		findmnt -lo TARGET | grep "${BASETMPDIR}" | xargs -r "${sudo}" umount
```

Comment:
Do you think we should turn off the warning here or not use `xargs`? Should we make a point of not using `xargs` and instead:
```sh
for mount in $( findmnt -lo TARGET | grep "${BASETMPDIR}" ); do
    ${sudo} umount "${mount}"
done
```
or something similar for read-ability?